.github/workflows/lint-pr.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Lint PR'
+
+on:
+  pull_request_target:
```

Comment:
Why `pull_request_target`? This has a lot of security implications.

Reply:
I missed this, sorry! Could you swnd a PR?